source/Octostache/Templates/TemplateParser.cs

```diff
@@ -122,12 +212,12 @@ static Parser<T> FollowedBy<T>(this Parser<T> parser, string lookahead)
                 if (!result.WasSuccessful)
                     return result;
 
-                if (result.Remainder.Position >= (i.Source.Length - lookahead.Length))
```

Comment:
Hmm... is it simplifying brackets in all situations?

In this specific line, it's simple enough so I wouldn't mind, but in other scenarios it can help with readability, personally.

Reply:
I think this one should be suppressed. Without the parentheses, I don't quickly know the precedence of the `>=` vs `-` operators.